### DISCUSSION TITLE
knowledge/blog/rl-series---part-1-reinforcement-learning---or-the-art-of-decision-making-in-an-uncertain-world/

### DISCUSSION THREAD

**Discussion by giscus** [TickLabVN/knowledges#25] [Q&A]
# knowledge/blog/rl-series---part-1-reinforcement-learning---or-the-art-of-decision-making-in-an-uncertain-world/

Summary of knowledge-sharing content during the research and project development process across departments at TickLab.

https://ticklabvn.github.io/knowledge/blog/rl-series---part-1-reinforcement-learning---or-the-art-of-decision-making-in-an-uncertain-world/

**Comment by ngyngcphu** [comment]
Nice work, hope part 2.
